Twig/GoogleAnalyticsTwigExtension.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Becklyn\RadBundle\Twig;
+
+use Becklyn\RadBundle\GoogleAnalytics\GoogleAnalyticsHelper;
+
+
+/**
+ *
+ */
+class GoogleAnalyticsTwigExtension extends AbstractTwigExtension
+{
+    /**
+     * @var GoogleAnalyticsHelper
+     */
+    private $analyticsHelper;
+
+
+    /**
+     * GoogleAnalyticsTwigExtension constructor.
+     *
+     * @param GoogleAnalyticsHelper $analyticsHelper
+     */
+    public function __construct (GoogleAnalyticsHelper $analyticsHelper)
+    {
+        $this->analyticsHelper = $analyticsHelper;
+    }
+
+
+
+    /**
+     * Renders all blocks of code required to initialize Google Analytics tracking
+     *
+     * @param string $trackingCode
+     *
+     * @return string
+     */
+    public function analyticsTracking (string $trackingCode)
+    {
+        if (empty($trackingCode))
+        {
+            return "";
+        }
+
+        $analyticsScript = $this->analyticsHelper->render();
+        if (empty($analyticsScript))
+        {
+            return "";
+        }
+
+        return sprintf("<script type='text/javascript'>%s; ga('create', '%s', 'auto'); ga('set', 'anonymizeIp', true); ga('send', 'pageview');</script>", $analyticsScript, $trackingCode);
```

Comment:
It's not necessary to go to the disc for the code. Just enter it here statically via NOWDOC and remove both the other service and the JS file.